tools/internal_ci/helper_scripts/prepare_build_linux_perf_rc

```diff
@@ -19,4 +19,23 @@
 ulimit -n 32768
 ulimit -c unlimited
 
+# Remove this nbefore merging. Only used for testing
+KOKORO_GITHUB_PULL_REQUEST_NUMBER=12552
+
+# Performance PR testing needs GH API key and PR metadata to comment results
+if [ -n "$KOKORO_GITHUB_PULL_REQUEST_NUMBER" ]; then
+  set +x
+  sudo apt-get install -y jq
+  ghprbTargetBranch=$(curl -s https://api.github.com/repos/grpc/grpc/pulls/$KOKORO_GITHUB_PULL_REQUEST_NUMBER | jq -r .base.ref)
+
+  gsutil cp gs://grpc-testing-secrets/github_credentials/oauth_token.txt ~/
+  export JENKINS_OAUTH_TOKEN=$(cat ~/oauth_token.txt)
```

Comment:
nit: it's not a JENKINS oauth token anymore, so maybe we can add a TODO somewhere to rename. I think it's GITHUB_OAUTH_TOKEN

Reply:
TODO added. There's also a lot of other Jenkins specific variables/naming conventions in our scripts that will need to be cleaned up after we're fully migrated.